### DISCUSSION TITLE
Figma Desktop App version 116.10.8，点击消息按钮后出错，不知是否是客户端原因

### DISCUSSION THREAD

**Discussion by leio1109** [Moonvy/figmaEX#35] [General]
如图：<img width="264" alt="image" src="https://github.com/Moonvy/figmaEX/assets/38681432/52f0ecdd-fe28-46cd-bcf0-b879da576f29">
<img width="248" alt="image" src="https://github.com/Moonvy/figmaEX/assets/38681432/1fd21772-cee9-4739-800b-41114a87dd79">
<img width="925" alt="image" src="https://github.com/Moonvy/figmaEX/assets/38681432/84edeb38-b3da-4036-9b6e-64225b839084">

**Comment by yArna** [comment]
这个与 FigmaEX 客户端没有关系
